pkg/metrics/sketch_series_test.go

```diff
@@ -4,67 +4,68 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/DataDog/datadog-agent/pkg/config"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
 	"github.com/DataDog/datadog-agent/pkg/quantile"
 
 	"github.com/DataDog/agent-payload/gogen"
-	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
```

Comment:
Is it necessary to use both `assert` and `require`?

Reply:
It was already there, and I guess the developer had some reasons to do it like this

Reply:
`require` terminates the test if the condition is not met so, though similar, it's not the same thing.